EventListeners/EventManager.php

```diff
@@ -58,38 +58,38 @@ public function calculateBestSellers(BestSellersEvent $event)
                 /** @var PdoConnection $con */
                 $con = Propel::getConnection();
 
-                $query = "
+                $statusList = BestSellers::getConfigValue('order_types');
```

Comment:
Please provide a default value here.

Reply:
I've added a check which sets a default value if it fails.